src/styles/global.scss

```diff
@@ -35,7 +34,6 @@ body {
   min-height: 100%;
   margin: 0;
   padding: 0;
-  background-color: color('sky', 'light');
```

Comment:
Thanks to the scope of the `ThemeProvider`, we can no longer style the background color or text color on the `body`, since it will not be theme-able. We've move these styles to a `div` on the `ThemeProvider`.